apps/selvbetjening-ui/src/inntektsjustering/2-inntekt-til-neste-aar/InntektsjusteringInntektTilNesteAar.tsx

```diff
@@ -1,151 +1,48 @@
-import { Accordion, Heading, HStack, TextField, VStack } from '@navikt/ds-react'
+import { HStack, VStack } from '@navikt/ds-react'
 import { SkjemaHeader } from '../../common/skjemaHeader/SkjemaHeader.tsx'
-import { useForm } from 'react-hook-form'
-import { NavigasjonMeny } from '../../common/NavigasjonMeny/NavigasjonMeny.tsx'
 import { InntektsjusteringInntektTilNesteAar as InntektsjusteringInntektTilNesteAarInnhold } from '../../sanity.types.ts'
 import { useSanityInnhold } from '../../common/sanity/useSanityInnhold.ts'
 import { useSpraak } from '../../common/spraak/SpraakContext.tsx'
 import { Navigate } from 'react-router-dom'
 import { SanityRikTekst } from '../../common/sanity/SanityRikTekst.tsx'
-import { SumAvOppgittInntekt } from './SumAvOppgittInntekt.tsx'
-
-export interface InntektTilNesteAarSkjema {
-    arbeidsinntektINorge: number
-    naeringsinntekt: number
-    AFPInntekt: number
-    alleInntekterIUtland: number
-}
-
-const inntektTilNesteAarDefaultValues: InntektTilNesteAarSkjema = {
-    arbeidsinntektINorge: 0,
-    naeringsinntekt: 0,
-    AFPInntekt: 0,
-    alleInntekterIUtland: 0,
-}
+import { Alder } from '../../types/person.ts'
+import { AttenTilFemtiSeksAarSkjema } from './skjemaer/AttenTilFemtiSeksAarSkjema.tsx'
+import { finnAlder } from './finnAlder.ts'
 
 export const InntektsjusteringInntektTilNesteAar = () => {
     const spraak = useSpraak()
 
-    const { innhold, error, isLoading } = useSanityInnhold<InntektsjusteringInntektTilNesteAarInnhold>(
+    const {
+        innhold,
+        error: innholdError,
+        isLoading: innholdIsLoading,
+    } = useSanityInnhold<InntektsjusteringInntektTilNesteAarInnhold>(
         '*[_type == "inntektsjusteringInntektTilNesteAar"]'
     )
 
-    const { register, watch, setValue } = useForm<InntektTilNesteAarSkjema>({
-        defaultValues: inntektTilNesteAarDefaultValues,
-    })
-
-    if (error && !isLoading) {
+    if (innholdError && !innholdIsLoading) {
         return <Navigate to="/system-utilgjengelig" />
     }
 
+    const velgSkjemaForInntekt = (alder: Alder) => {
+        switch (alder) {
+            case Alder.ATTEN_TIL_FEMTI_SEKS:
+                return <AttenTilFemtiSeksAarSkjema />
+        }
```

Comment:
Kanskje litt unødvendig med en switch for ett case...? 
Noe fortsettelse følger her, eller?

Reply:
Kommer 3 skjemaer til utifra alder på personen som skal sende inn inntekt